app/middleware/auth.ts

```diff
@@ -9,10 +10,19 @@ export const authMiddleware = (req: express.Request, res: express.Response, next
             return res.status(401).json({message: 'Not Authorized'});
         }
         jwt.verify(token, jwtKey);
+        
+        const base64Url = token.split('.')[1];
+        const base64 = base64Url.replace('-', '+').replace('_', '/');
+        const payload = JSON.parse(Buffer.from(base64, 'base64').toString('binary'));
+        
+        res.locals.auctioneer = payload.auctioneer;
+        res.locals.email = payload.email;
```

Comment:
This will just pull the requestor's email and auctioneer status from the jwt sent in the request.